driver/src/main/java/org/neo4j/driver/internal/security/TLSSocketChannel.java

```diff
@@ -372,6 +372,8 @@ private HandshakeStatus wrap( ByteBuffer buffer ) throws IOException, ClientExce
                 cipherOut.compact();
             }
             break;
+        case CLOSED:
```

Comment:
Just wonder why this is needed? Is it a fix to something?

Reply:
Added stress test closes all connections to simulate them breaking. Without this branch closing might result in `ClientException`, which is not retriable. It feels better to have it as `IOException` so that it bubbles up to users as `ServiceUnavailableException` or `SessionExpiredException`. Does this sound sensible?